lib/chef-dk/builtin_commands.rb

```diff
@@ -15,46 +15,48 @@
+                         desc: "Prints environment variables used by #{ChefDK::Dist::PRODUCT}"
 
   c.builtin "gem", :GemForwarder, require_path: "chef-dk/command/gem",
-                                  desc: "Runs the `gem` command in context of the embedded ruby"
+                                  desc: "Runs the `gem` command in context of the the embedded Ruby"
 
-  c.builtin "generate", :Generate, desc: "Generate a new app, cookbook, or component"
+  c.builtin "generate", :Generate, desc: "Generate a new repository, cookbook, or other component"
 
-  c.builtin "shell-init", :ShellInit, desc: "Initialize your shell to use ChefDK as your primary ruby"
+  c.builtin "shell-init", :ShellInit, desc: "Initialize your shell to use #{ChefDK::Dist::PRODUCT} as your primary Ruby"
 
   c.builtin "install", :Install, desc: "Install cookbooks from a Policyfile and generate a locked cookbook set"
 
   c.builtin "update", :Update, desc: "Updates a Policyfile.lock.json with latest run_list and cookbooks"
 
-  c.builtin "push", :Push, desc: "Push a local policy lock to a policy group on the server"
+  c.builtin "push", :Push, desc: "Push a local policy lock to a policy group on your #{ChefDK::Dist::SERVER_PRODUCT}"
 
-  c.builtin "push-archive", :PushArchive, desc: "Push a policy archive to a policy group on the server"
+  c.builtin "push-archive", :PushArchive, desc: "Push a policy archive to a policy group on your #{ChefDK::Dist::SERVER_PRODUCT}"
 
-  c.builtin "show-policy", :ShowPolicy, desc: "Show policyfile objects on your Chef Infra Server"
+  c.builtin "show-policy", :ShowPolicy, desc: "Show policyfile objects on your #{ChefDK::Dist::SERVER_PRODUCT}"
 
   c.builtin "diff", :Diff, desc: "Generate an itemized diff of two Policyfile lock documents"
 
-  c.builtin "export", :Export, desc: "Export a policy lock as a Chef Zero code repo"
+  c.builtin "export", :Export, desc: "Export a policy lock as a #{ChefDK::Dist::ZERO_PRODUCT} code repo"
 
-  c.builtin "clean-policy-revisions", :CleanPolicyRevisions, desc: "Delete unused policy revisions on the server"
+  c.builtin "clean-policy-revisions", :CleanPolicyRevisions, desc: "Delete unused policy revisions on your #{ChefDK::Dist::SERVER_PRODUCT}"
 
-  c.builtin "clean-policy-cookbooks", :CleanPolicyCookbooks, desc: "Delete unused policyfile cookbooks on the server"
+  c.builtin "clean-policy-cookbooks", :CleanPolicyCookbooks, desc: "Delete unused policyfile cookbooks on your #{ChefDK::Dist::SERVER_PRODUCT}"
 
-  c.builtin "delete-policy-group", :DeletePolicyGroup, desc: "Delete a policy group on the server"
+  c.builtin "delete-policy-group", :DeletePolicyGroup, desc: "Delete a policy group on your #{ChefDK::Dist::SERVER_PRODUCT}"
 
-  c.builtin "delete-policy", :DeletePolicy, desc: "Delete all revisions of a policy on the server"
+  c.builtin "delete-policy", :DeletePolicy, desc: "Delete all revisions of a policy on your #{ChefDK::Dist::SERVER_PRODUCT}"
```

Comment:
Nit: "your" might not apply, in cases like hosted.  Maybe "the"?

Reply:
There was a mix of really professional vs. more personal wording in the help messaging. I'm curious where @chef/docs-team thinks we should go with it. This one probably makes more sense as "the", but our tone is all over the place.